metadata/blinky.ukp.informatik.tu-darmstadt.de%2Fshibboleth.xml

```diff
@@ -0,0 +1,135 @@
+    <alg:DigestMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2000/09/xmldsig#sha1"/>
+    <alg:SigningMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2001/04/xmldsig-more#ecdsa-sha512"/>
+    <alg:SigningMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2001/04/xmldsig-more#ecdsa-sha384"/>
+    <alg:SigningMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2001/04/xmldsig-more#ecdsa-sha256"/>
+    <alg:SigningMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2001/04/xmldsig-more#ecdsa-sha224"/>
+    <alg:SigningMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2001/04/xmldsig-more#rsa-sha512"/>
+    <alg:SigningMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2001/04/xmldsig-more#rsa-sha384"/>
+    <alg:SigningMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"/>
+    <alg:SigningMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2009/xmldsig11#dsa-sha256"/>
+    <alg:SigningMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2001/04/xmldsig-more#ecdsa-sha1"/>
+    <alg:SigningMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2000/09/xmldsig#rsa-sha1"/>
+    <alg:SigningMethod xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" Algorithm="http://www.w3.org/2000/09/xmldsig#dsa-sha1"/>
+  </md:Extensions>
+    <md:SPSSODescriptor protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
+        <md:Extensions>
+            <mdui:UIInfo xmlns:mdui="urn:oasis:names:tc:SAML:metadata:ui">
+                <mdui:DisplayName xml:lang="en">INCEpTION - Community Server</mdui:DisplayName>
+                <mdui:Description xml:lang="en">INCEpTION provides a semantic annotation platform offering intelligent annotation assistance and knowledge management.</mdui:Description>
+                <mdui:DisplayName xml:lang="de">INCEpTION - Community Server</mdui:DisplayName>
+                <mdui:Description xml:lang="de">INCEpTION ist eine semantische Annotationsplattform mit intelligenter Annotationshilfe und Wissensmanagement.</mdui:Description>
+                <mdui:InformationURL xml:lang="en">https://inception-project.github.io/</mdui:InformationURL>
+                <mdui:Logo height="239" width="239">https://inception-project.github.io//assets/img/logo.png</mdui:Logo>
+                <mdui:Keywords xml:lang="en">INCEpTION annotation</mdui:Keywords>
+                <mdui:PrivacyStatementURL xml:lang="de">https://www.tu-darmstadt.de/datenschutzerklaerung.en.jsp</mdui:PrivacyStatementURL>
```

Comment:
Add `<mdui:PrivacyStatementURL xml:lang="en">` here as described in the [PR comment](https://github.com/clarin-eric/SPF-SPs-metadata/pull/197#issuecomment-617668564).